test/Prompt/ValuePrinter/Collections.C

```diff
@@ -0,0 +1,38 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+
+//------------------------------------------------------------------------------
+
+// RUN: cat %s | %cling -Xclang -verify 2>&1 | FileCheck %s
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+std::vector<bool> Bv(5,5)
+// FIXME: Printing std::vector<bool> is still broken.
+// But the line above at least tests cling doesn't crash because of it.
+// BROKENCHECK: (std::vector<bool> &) { true, true, true, true, true }
```

Comment:
No, it currently prints no `true` or `false` but "unrepresentable value". But indeed, at least it doesn't crash anymore. If we want proper value printing of `vector<bool>` (`vector` of everything else works) then we'll need to `#include <vector>` in RuntimePrintValue.h.